data/json/mapgen/bank.json

```diff
@@ -221,6 +221,7 @@
       "computers": {
         "6": {
           "name": "High Security Consolidated Computerized Bank of the Treasury",
+          "access_denied": "ERROR!  Access denied! Unauthorized access will be met with lethal force!",
```

Comment:
These messages needs to be extracted for translation. You should update `extract_mapgen` in `lang/extract_json_strings.py` to do so.

Reply:
I've used `update_pot.sh` to test my `extract_json_strings.py` changes. Am I supposed to commit the updated pot file or should I leave that alone? I get the impression from https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/TRANSLATING.md#maintainers that only maintainers should do that.

Reply:
Yeah, leave the pot file as is. It will be updated in a separate PR.